tutorials/clustering.md

```diff
@@ -13,17 +12,25 @@ In this tutorial, we will show how you can create a distributed hypertable withi
 a distributed database. We will set up the distributed database with one primary access node
 and two data nodes across three distinct TimescaleDB instances.
 
-Prerequisites:
-- 3 instances of TimescaleDB installed (private BETA branch only)
-- A superuser set up on all 3 instances of TimescaleDB
-- Network connectivity between the 3 TimescaleDB instances
-- Must be on PG11
+It is assumed before starting this tutorial you will have downloaded and installed the
+TimescaleDB multinode beta package on three networked machines running Postgres 11. It's
+also assumed you'll have access to some superuser role common to these three machines.
+
+Beyond this, please ensure that you've completed the following steps. Some or most of
+these may not be necessary depending on how you initially set up your instances (the 
+docker image should be have most of this already configured):
+- Make sure you've enabled 'max_prepared_transactions' on all the instances (set this to 100 if you're unsure), this parameter is located in `postgresql.conf`
+- Make sure the data nodes are properly listening for network traffic. `postgresql.conf` should have a `listen_addresses` value including the access node's IP. If the field is missing or `localhost`, change it to the needed IP, or `"*"`
+- The superuser role must have an entry in the `pg_hba.conf` file. This should look something like: ```host    all    <superuser role>    <connection source>    trust``` The connection source should be the `<access node's IP>/32` or some encompasing range, or `samenet` if the instances are on the same subnet. For using authentication mechanism's other than `trust`, please see the [authentication guide][data-node-authentication]. Note that if the instance is specifying something other than `pg_hba.conf` in the `hba_file` field of `postgresql.conf`, then that is the file which must have the above line.
+
+After making any of these changes, please restart your postgres instance to ensure the
+new settings take effect.
 
 ## Define the topology for your distributed database
 
 Now that you have 3 running instances of TimescaleDB, you can go ahead and set up your
-distributed database. Connect to the instance that you would like to act as the access node,
-and create a new database there:
+distributed database. Connect to the instance that you would like to act as the access 
+node using your superuser role, and create a new database there:
```

Comment:
May be add a superuser to the connection command below, e.g.:
```
\c multinode superuser
```

Reply:
done